google/cloud/pubsub/internal/publisher_tracing_connection.cc

```diff
@@ -54,7 +54,8 @@ opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span> StartPublishSpan(
                                    static_cast<std::int64_t>(MessageSize(m))},
                               },
                               options);
-
+  span->SetAttribute("cloud-cxx.function",
```

Comment:
shouldn't we move this up two lines?

Reply:
Yeah that would be better.

bleep bloop... the other option seemed easier at the time of writing. changed.